centreon/tests/e2e/features/Dashboards/01-dashboard-creation/index.ts

```diff
@@ -195,3 +195,38 @@ Then('the form fields are empty', () => {
   cy.getByLabel({ label: 'Name', tag: 'input' }).should('be.empty');
   cy.getByLabel({ label: 'Description', tag: 'textarea' }).should('be.empty');
 });
+
+Given(
+  "a dashboard with existing widgets in the dashboard administrator user's dashboard library",
+  () => {
+    cy.insertDashboardWithPatch(dashboards.fromDashboardCreatorUser,patchBody);
+    cy.visit(`${Cypress.config().baseUrl}/centreon/home/dashboards`);
+  }
+);
+
+
+When('the dashboard administrator user starts to edit the dashboard', () => {
+      cy.getByLabel({ label: 'view', tag: 'button' }).click();
+});
+
+Then("creates a new dashboard on the previous dashboard's edition page", () => {
+      cy.getByTestId({ testId: 'ArrowDropDownIcon' }).click();
+      cy.getByTestId({ testId: 'AddIcon' }).click();
+      cy.getByLabel({ label: 'Name', tag: 'input' }).type(dashboards.default.name);
+      cy.getByLabel({ label: 'Description', tag: 'textarea' }).type(dashboards.default.description);
+      cy.getByLabel({ label: 'Create', tag: 'button' }).click()
+      cy.wait('@createDashboard')
+
+});
+
+Then("the user is redirected to the newly created dashboard's edition page", () => {
+  cy.url().should((url) => {
+    const urlStr = url.toString();
+    expect(urlStr).to.include('/dashboard');
+    expect(urlStr).to.include('edit=true');
+  });
+});
+
+Then("the newly created dashboard is empty", () => {
+  cy.getByTestId({ testId: 'AddIcon' }).should('be.visible')
```

Comment:
I am not sure the assertion does what is corresponding to the step edition
I guess there should be something like `should('not.be.visible')` or `should('not.exist')`

Reply:
i think the object referred to with the AddIcon testId only appears whenever there'sq no widget on the dashboard, Wassim should confirm this but it probably has its place there

Reply:
There are mutiple AddIcon in the page, so this assertion is not enough : 
![image](https://github.com/centreon/centreon/assets/11978823/aa1f4451-652a-44a3-b263-ae417a2da9cb)